packages/core/src/Stamp/Stamp.stories.js

```diff
@@ -87,3 +87,67 @@ storiesOf('Stamp', module)
       </Stamp>
     </div>
   ))
+  .add('Pass an array of sizes', () => (
+    <div>
+      <Stamp
+        borderColor='blue'
+        bg='blue'
+        color='white'
```

Comment:
Could we switch these to use palette colours instead?
```suggestion
        color='primary'
```

Reply:
I'm not sure exactly how it's supposed to work, but this is what `color='primary'` looks like on Stamp (far left):
<img width="367" alt="Screen Shot 2020-11-06 at 3 55 09 PM" src="https://user-images.githubusercontent.com/3260642/98413944-6fdc4800-2072-11eb-99c5-24d92d9cd005.png">

Reply:
@pcln-brianna , just saw your update. That looks as expected 👍

Reply:
Yeah I realized it was the reverse of what I was thinking after checking our docs haha. Tested the updated suggestion:
![Screen Shot 2020-11-06 at 2 57 35 PM](https://user-images.githubusercontent.com/12076625/98414162-7eaa0700-2040-11eb-9e11-2d92f1b0c0a6.png)